src/BiDirectionalLinkedListImp.h

```diff
@@ -0,0 +1,152 @@
+//
+// Created by shvmo on 02/05/2020.
+//
+
+#ifndef DATASTRUCTURESWET1_BIDIRECTIONALLINKEDLISTIMP_H
+#define DATASTRUCTURESWET1_BIDIRECTIONALLINKEDLISTIMP_H
+
+#include <cassert>
```

Comment:
i don't really care if we use asserts here but this is cpp we have exceptions 🤔

Reply:
Asserts are for programming uses and debugging and are gone once you compile. Without debug flag. Exceptions are differnet

Reply:
Exceptions are used for users who use your stuff and deal with runtime errors that you can't control.